fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Field.java

```diff
@@ -64,6 +64,17 @@ public OneOfThem oneOfThem() {
         return new OneOfThem(fieldName);
     }
 
+    /**
+     * If the associated field is a repeated one, this allows you to match against one of those values, the record will
```

Comment:
This is a comma splice (as is the first sentence of the docs on `oneOfThem()`). (More specifically, the comma between "values" and "the" is splicing; the comma between "one" and "this" is fine.)